diff --git a/.github/workflows/tests.yml b/.github/workflows/tests.yml
index 597f136..156afb8 100644
--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -11,7 +11,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10']
+        python: ['3.7', '3.8', '3.9']
+        # Time to switch to pytest or nose2?
+        # nosetests is broken on 3.10
+        # AttributeError: module 'collections' has no attribute 'Callable'
+        # https://github.com/nose-devs/nose/issues/1099
         django:
         - 'Django>=2.2,<3.0'
         - 'Django>=3.0,<3.1'
@@ -19,7 +23,7 @@ jobs:
         - 'Django>=3.2,<3.3'
         experimental: [false]
         include:
-        - python: '3.10'
+        - python: '3.9'
         - django: 'https://github.com/django/django/archive/refs/heads/main.zip#egg=Django'
           experimental: true
           # NOTE this job will appear to pass even when it fails because of
